packages/pwa-kit-extension-sdk/src/configs/utils.ts

```diff
@@ -34,34 +34,53 @@ Handlebars.registerHelper('jsonStringify', (context) => JSON.stringify(context,
 // to issues with pathing because the current working directory for the loader isn't the same as the base project.
 // We can look to resolve this in the future as it would be nice to have a independant file for the template.
 const templateString = dedent`
+    import {getConfiguredExtensions} from '../../shared/utils/helpers'
     {{#if (isWeb @root.target)}}
     import loadable from '@loadable/component'
     {{/if}}
 
-    {{#each installed}}
+    {{#each configured}}
     {{#if (isNode @root.target)}}
-    import {{getInstanceName .}} from '{{.}}/setup-server'
+    import {{getInstanceName this.[0]}} from '{{this.[0]}}/setup-server'
     {{else}}
-    const {{getInstanceName .}}Loader = loadable.lib(() => import('{{.}}/setup-app'))
+    const {{getInstanceName this.[0]}}Loader = loadable.lib(() => import('{{this.[0]}}/setup-app'))
     {{/if}}
     {{/each}}
 
+    const imports = {
+        {{#each configured}}
+        {{#if (isNode @root.target)}}
+        '{{this.[0]}}': {{getInstanceName this.[0]}},
+        {{else}}
+        '{{this.[0]}}': {{getInstanceName this.[0]}}Loader,
+        {{/if}}
+        {{/each}}
+    }
+
     {{#if (isNode @root.target)}}
     const getApplicationExtensions = () => {
-        {{#if configured}}
-        return [{{#each configured}}new {{getInstanceName this.[0]}}({{{jsonStringify this.[1]}}}){{#if (isNotLast @index @root.configured.length)}}, {{/if}}{{/each}}]
-        {{else}}
-        return []
-        {{/if}}    
+        const configuredExtensions = getConfiguredExtensions()
```

Comment:
I propose that for a better developer workflow, we also parse the configured extension at runtime too.

So that developers can update the extensions config and simply reload the browser to see the effect. No need to restart their dev server.

Reply:
I'm ok with that.